src/components/Notification/InactivityMessage.jsx

```diff
@@ -69,21 +68,34 @@ const InactivityMessage = () => {
     localStorage.removeItem(`issuerConfig:${SOLID_IDENTITY_PROVIDER.slice(0, -1)}`);
   };
 
-  return (
-    <StyledModal open={showPopup && activeUser}>
-      <StyledContainer>
-        <p>You have been inactive for a few minutes now. Would you like to log out?</p>
-        <ButtonsContainer>
-          <StyledButton onClick={() => setShowPopup(false)}>Continue Session</StyledButton>
-          <LogoutButton>
-            <StyledLogoutButton onClick={handleLogout}>Log Out</StyledLogoutButton>
-          </LogoutButton>
-        </ButtonsContainer>
-      </StyledContainer>
-    </StyledModal>
-  );
+  return showPopup && activeUser ? (
```

Comment:
This is great already. I was wondering if there's a simpler alternative to this using just &&.